db/migrate/20170730154117_create_account_migrations.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccountMigrations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :account_migrations do |t|
+      t.integer :old_person_id, null: false
+      t.integer :new_person_id, null: false
+    end
+
+    add_foreign_key :account_migrations, :people, column: :old_person_id
+    add_foreign_key :account_migrations, :people, column: :new_person_id
+
+    add_index :account_migrations, %i[old_person_id new_person_id], unique: true
```

Comment:
Since we don't support this yet then we should probably add another unique index for `new_person_id`. We can remove this index if we introduce this feature afterwards, right? Also we have separate `unique` validation for new and old person in Rails now.